src/screens/Asset/ReceiveModal.js

```diff
@@ -206,20 +204,19 @@ class ReceiveModal extends React.Component<Props> {
             <Button
               title={t('button.buyTokens')}
               onPress={handleBuyTokens}
-              positive
+              primarySecond
               width={buttonWidth}
               small={needsSmallButtons}
-              regularText
               textStyle={{ paddingTop: 4 }}
+              block={false}
             />
             )}
             <Button
               title={t('button.shareAddress')}
               onPress={this.handleAddressShare}
-              width={buttonWidth}
+              width={showBuyTokensButton ? buttonWidth : undefined}
```

Comment:
How we can change from `undefined`?

Reply:
Hm, I guess it can be solved with flex rather than calculating the width.
On it :)